src/py_avro_schema/_schemas.py

```diff
@@ -567,6 +567,14 @@ def data(self, names: NamesType) -> JSONObj:
             "items": self.items_schema.data(names=names),
         }
 
+    def make_default(self, py_default: collections.abc.Sequence) -> collections.abc.Sequence:
+        """Return an Avro schema compliant default value for a given Python Sequence
+
+        :param py_default: The Python sequence to generate a default value for.
+        """
+        origin = get_origin(self.py_type) or self.py_type
+        return origin(self.items_schema.make_default(item) for item in py_default)
```

Comment:
Just wondering, should this not always return a Python list? And the return type should then be a `JSONArray`?

Reply:
You are right, considering the avro spec (the data func also did this. this confused me a lot because then assert_schema prints expected and actual looking identically, but the dict representations have been different. Hard to debug and changed)